sdk/core/core-autorest/package.json

```
@@ -0,0 +1,122 @@
+  "sdk-type": "client",
+  "main": "dist/index.js",
+  "module": "dist-esm/src/index.js",
+  "types": "types/latest/core-autorest.d.ts",
+  "typesVersions": {
+    "<3.6": {
+      "types/latest/src/*": [
+        "types/3.1/src/*"
+      ]
+    }
+  },
+  "scripts": {
+    "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
+    "build:browser": "npm run build:ts && cross-env ONLY_BROWSER=true rollup -c 2>&1",
+    "build:node": "npm run build:ts && cross-env ONLY_NODE=true rollup -c 2>&1",
+    "build:samples": "cd samples && tsc -p .",
+    "build:test": "npm run build:ts && npm run bundle:test",
+    "build:test:browser": "npm run build:ts && npm run bundle:test:browser",
+    "build:test:node": "npm run build:ts && npm run bundle:test:node",
+    "build:ts": "tsc -p .",
+    "build:types": "downlevel-dts types/latest/ types/3.1/",
+    "build": "npm run build:ts && rollup -c 2>&1 && api-extractor run --local && npm run build:types",
+    "bundle:test": "rollup -c rollup.test.config.js 2>&1",
+    "bundle:test:browser": "cross-env ONLY_BROWSER=true rollup -c rollup.test.config.js 2>&1",
+    "bundle:test:node": "cross-env ONLY_NODE=true rollup -c rollup.test.config.js 2>&1",
+    "check-format": "prettier --list-different \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "clean": "rimraf dist dist-* types *.tgz *.log",
+    "execute:samples": "echo skipped",
+    "extract-api": "npm run build:ts && api-extractor run --local",
+    "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "integration-test:browser": "echo skipped",
+    "integration-test:node": "echo skipped",
+    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
+    "lint:fix": "eslint package.json tsconfig.json api-extractor.json src test --ext .ts --fix --fix-type [problem,suggestion]",
+    "lint": "eslint package.json tsconfig.json api-extractor.json src test --ext .ts -f html -o coreAutoRest-lintReport.html || exit 0",
+    "pack": "npm pack 2>&1",
+    "prebuild": "npm run clean",
+    "test:browser": "npm run build:test:browser && npm run unit-test:browser && npm run integration-test:browser",
+    "test:node": "npm run build:test:node && npm run unit-test:node && npm run integration-test:node",
+    "test": "npm run clean && npm run build:ts && npm run bundle:test:node && npm run unit-test:node && npm run bundle:test:browser && npm run unit-test:browser && npm run integration-test:node && npm run integration-test:browser",
+    "unit-test:browser": "karma start --single-run",
+    "unit-test:node": "mocha --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js dist-test/index.node.js",
+    "unit-test": "npm run unit-test:node && npm run unit-test:browser"
+  },
+  "files": [
+    "dist/",
+    "dist-esm/src/",
+    "types/",
```

I think we can exclude the non-rolledup types that are in types/src and types/test
```suggestion
    "types/latest/",
    "types/3.1/"
```

---

I changed this slightly to only include the rolled up versions